sea-orm-macros/Cargo.toml

```diff
@@ -21,3 +21,7 @@ syn = { version = "^1", default-features = false, features = [ "full", "derive",
 quote = "^1"
 heck = "^0.3"
 proc-macro2 = "^1"
+
+[dev-dependencies]
+sea-orm = { path = "../", features = ["macros"] }
+serde = { version = "^1.0", features = ["derive"] }
```

Comment:
Hey @charleschege, I have fixed all the errors inside `sea-orm` doctest on fcf3ea9. And I know why `cargo test --doc` passes while `cargo test --workspace` fails.

In `sea-orm-macros/src/lib.rs`, you have include some code snippets that requires `sea-orm`. However, `sea-orm-macros` does not depends on `sea-orm`. So, all of the doctests failed. By adding `dev-dependencies` will solve the problem. 6904b9f

Reply:
Thanks! This has resolved all the issues I had when testing documentation examples.